var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -144,7 +144,7 @@ class Seacas(CMakePackage):
     depends_on("netcdf-c@4.8.0:~mpi", when="~mpi")
     depends_on("hdf5+hl~mpi", when="~mpi")
 
-    depends_on("fmt@10.1.0", when="@2023-10-24:")
+    depends_on("fmt@10.1.0", when="@2023-11-27:")
```

Comment:
Have you also considered loosening the version requirements? It should be fine with all `fmt@10` versions I would think. Overconstrained versions can cause problems when integrating into a larger software ecosystem.

Reply:
IIRC, there was a problem with 10.0 which was triggered by either the intel or nvidia compiler which was fixed in 10.1.  I will verify that and if I am remembering correctly, will loosen the requirement in a subsequent commit.